RELEASE-NOTES.md

```diff
@@ -96,6 +96,7 @@ All of the above apply to:
 This includes API changes we did not warn about since at least `3.11.0` (2021-01).
 
 - Setting initial values through `pm.Distribution(testval=...)` is now `pm.Distribution(initval=...)`.
+- In favour of `sigma`, the deprecated `sd` has been removed.
```

Comment:
```suggestion
- Alternative `sd` keyword argument has been removed from all distributions. `sigma` should be used instead
```

Reply:
Should add PR link after description. You can see how that's done in other entries of the release notes

Reply:
Done, thanks : )